index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>porfolio project</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+ <div class="header-section">
+<a href="#" class="logo">My Logo</a>
+<nav> <img src="images/Union.png" alt=""></nav>
```

Comment:
- Kindly fix the hamburger image path. It currently is not appearing. It is important to show this icon since its what you will use in mobile devices to  navigate.
- Also i suggest you remove the underline from the logo and use the same font as figma template. The purpose of this project is to show that given a design you can replicate it.
This is how your page currently appears
<img width="1222" alt="Screenshot 2023-01-19 at 15 35 17" src="https://user-images.githubusercontent.com/48546992/213445385-9b7c2fb2-1b11-4727-967f-d08c9a4b7764.png">
This is the template 
<img width="381" alt="Screenshot 2023-01-19 at 09 40 33" src="https://user-images.githubusercontent.com/48546992/213445733-545b60ac-86f9-4b99-a7bc-ce4f4b3a0287.png">